docs-devsite/vertexai-preview.schemaparams.md

```diff
@@ -0,0 +1,21 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# SchemaParams interface
+User-facing params passed to specific Schema static methods.
```

Comment:
By "user", do you mean end-user?

Reply:
Developer, as in, not internal and only used by other internal code. I can reword this to not be about who it's facing, but about if you're using the `Schema` convenience classes or looking at the raw request format.